src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+const bigImg = document.getElementById('largeImg');
```

Comment:
Ensure that there is an element with the ID 'largeImg' in your HTML. Otherwise, 'bigImg' will be null, and attempting to set attributes on it will cause an error.